src/components/VAudioTrack/VGlobalAudioTrack.vue

```diff
@@ -154,7 +154,6 @@ export default defineComponent({
         audio.addEventListener('timeupdate', setTimeWhenPaused)
         audio.addEventListener('durationchange', setDuration)
         currentTime.value = audio.currentTime
-        duration.value = audio.duration
```

Comment:
I guess it would be covered by `durationchange` and `setDuration` once the file is finished loading?

I wonder if this has to do with some providers not sending the content length for the preview audio, so then the audio object only has a coherent duration once the file is actually loaded and the calculation has been made :thinking:

Reply:
Oh, that might be the reason!

Reply:
I tried different audios from Jamendo and Freesound, and all of them have the duration of NaN here initially. If you click on the same audio for the second time (play one audio - then play a different audio - then go back to the first one and click play), the duration is correct.